Pipfile

```diff
@@ -18,6 +18,11 @@ matplotlib = "*"
 pytest = "*"
 pytest-timeout = "*"
 pytest-cov = "*"
+hypothesis-auto = "*"
+hypothesis = "*"
+pytest-mypy = "*"
+mypy = "*"
+pre-commit = "==2.0.1"
```

Comment:
why is pre-commit pinned down to this specific version?

Reply:
It should not be pinned, nice catch!!